data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -ex
+
+IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic:master"}
+IRONIC_INSPECTOR_IMAGE=${IRONIC_INSPECTOR_IMAGE:-"quay.io/metal3-io/ironic-inspector:master"}
+IPA_DOWNLOADER_IMAGE=${IPA_DOWNLOADER_IMAGE:-"quay.io/metal3-io/ironic-ipa-downloader:master"}
+COREOS_DOWNLOADER_IMAGE=${COREOS_DOWNLOADER_IMAGE:-"quay.io/openshift-metal3/rhcos-downloader:master"}
```

Comment:
Maybe add a comment here noting that all of these will be changed to get image specs from the release image, once all of these images are included there?  We should link to a bug tracking that.

Reply:
Ack, good point, I'll raise an issue similar to #2067 to track this

Reply:
https://github.com/openshift/installer/issues/2090 raised